src/ExceptionConverter.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\FtpExtractor;
+
+use Keboola\Component\UserException;
+use League\Flysystem\FileNotFoundException;
+
+final class ExceptionConverter
```

Comment:
Chybí test.

Reply:
S tím testem jsem trochu váhal, zda je to za vás třeba, Jasně že je, tak super :)

Reply:
Pokud je to funkcionální, tak to může mít test a tak by ho to mělo mít. Pokud je to pain a muselo by se to mockovat, tak je OK se na to vyprdnout :)

Reply:
Vidím to stejně, díky za upřesnění